src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -66,7 +67,13 @@ public class ExceptionlessSink : ILogEventSink {
         public ExceptionlessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true, ExceptionlessClient client = null) {
             _additionalOperation = additionalOperation;
             _includeProperties = includeProperties;
-            _client = client ?? ExceptionlessClient.Default;
+            if (client != null) {
+                _client = client;
+            }
+            else {
+                _client = ExceptionlessClient.Default;
+                _client.Configuration.UseLogger(new SelfLogLogger());
```

Comment:
Do you think we should only do this on the default instance instead of every time? If you pass in your own client instance (you might be passing in ExceptionlessClient.Default), we won't wire up the self log? Maybe do _client = client ?? ExceptionlessClient.Default; and the following line call useLogger